### ISSUE TITLE
Do you have .NET and .NET core images for Kata container

### ISSUE THREAD

**Issue by VenkateshSrini** [kata-containers/community#101]
Hi,
Is it possible to execute .NET on Kata container ?

Do you have a .NET core base image with Kata container ?

**Comment by jodh-intel** [comment]
Hi @VenkateshSrini. We don't have such an image currently. You could potentially create one yourself using [`osbuilder`](https://github.com/kata-containers/osbuilder). However, to help us understand this question, please could you provide more information on what such an image would contain?

**Comment by grahamwhaley** [comment]
Hi @VenkateshSrini - Kata runs standard OCI images, like Docker images - it does not (generally) have, or need, its own special or modified images. It is a drop in replacement for the standard image runtimes (like `runc` under Docker or Kubernetes).
Kata only currently supports Linux images though. I presume you are looking for Linux based .NET image support. If you are asking after Windows based images, then no, Kata does not currently support them (or any other Windows image) at present.

A quick surf turned up .NET Linux images on the Docker Hub I think: https://hub.docker.com/_/microsoft-dotnet-core
afaik, nobody has tried them under Kata (or at least not reported any problems with running them if they have).

**Comment by VenkateshSrini** [comment]
That brings another question to my mind. How can I install Kata in window 10 machine like a Docker CE. If that is the case then we can quickly check .NET core images. 

Another point is Microsoft has Windows Nano server, which is also an Hyper V container. Will it possible to host that on Kata runtime

**Comment by grahamwhaley** [comment]
Hi @VenkateshSrini - The Kata binaries (the bits you need on your host system) are also Linux only at this point in time. So, today, Kata only runs Linux containers on Linux systems afaik.
I think there is little technically stopping anybody porting the relevant bits of code over to support other host and container operating systems, but nobody has attempted it or is working on it at the moment. One of the key items would be to port the Kata Agent over to target guest (container) environment - to get a feel for what the Agent is and does, the [architecture documentation](https://github.com/kata-containers/documentation/blob/master/design/architecture.md#overview) is probably a good place to start.

In summary then, if you want to run MS .NET Linux based images on a Linux host, then maybe that works for Kata. If any parts (host or guest) are not Linux based, afaik that will not work today. I hope that helps clarify?

**Comment by VenkateshSrini** [comment]
Thanks
